Bottom: 042d8cf1bc87b95102528916350e92b145e8328b
Top:    55cb084f4b751c390b0b8e1b445463fecd349168
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 18:00:38 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 1970edc..d53235a 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2177,6 +2177,7 @@ int main(int argc, char *argv[])
 
 		lookup_nss_read_master(master_list, 0);
 		master_show_mounts(master_list, dump);
+		master_kill(master_list);
 		if (dump)
 			free((void *) dump);
 		exit(0);
